test/mysql_test.cpp

```diff
@@ -100,6 +100,8 @@ TEST_CASE_METHOD(mysql_fixture, "catalog_tables_test", "[mysql][catalog][tables]
     catalog_tables_test();
 }
 
+// TODO: Add catalog_table_privileges_test - SQLTablePrivileges returns empty result set
```

Comment:
I'd be ok with noting this issue in the documentation somewhere and omitting this test in MySQL.